clio-server/src/main/scala/org/broadinstitute/clio/server/dataaccess/SystemElasticsearchDAO.scala

```diff
@@ -25,13 +26,12 @@ trait SystemElasticsearchDAO extends SearchDAO { this: HttpElasticsearchDAO =>
     }
   }
 
-  override def initialize(): Future[Unit] = {
-    for {
-      _ <- waitForSearchReady()
-      _ <- createOrUpdateIndex(ElasticsearchIndex.WgsUbam)
-      _ <- createOrUpdateIndex(ElasticsearchIndex.Gvcf)
-
-    } yield ()
+  override def initialize(
+    indexes: immutable.Seq[ElasticsearchIndex[_]]
```

Comment:
This is an `immutable.Seq`, instead of just a `Seq`, because `Future.foldLeft` expects an `immutable.Iterable`.